radicle-node/src/service/message.rs

```diff
@@ -271,6 +320,14 @@ pub struct Announcement {
 }
 
 impl Announcement {
+    /// Proof-of-work iterations for announcements.
+    #[cfg(test)]
+    pub const POW_PARAMS: (u8, u32, u32) = (1, 1, 1);
+    #[cfg(not(test))]
+    pub const POW_PARAMS: (u8, u32, u32) = (15, 8, 1);
```

Comment:
Is there a simple way to explain why the numbers are chosen?  Coming from the view of someone new to scrypt, and understanding when and how we should change them.

Reply:
Yeah I can add some docs :+1: